shell/shared/imageLoader/mac/ImageLoaderMac.mm

```diff
@@ -21,8 +21,14 @@
   if ([nsFileName length] == 0) {
     return nil;
   }
+  NSArray * parts = [nsFileName componentsSeparatedByString: @"."];
+  NSString *name = parts[0];
+  NSString *suffix = parts[1];
+  NSString* path = [@"images/" stringByAppendingString:name];
```

Comment:
This doesn't look correct, at first glance. Or at least not complete, since not all platforms have this hardcoded references to "images" right now.

Can you please elaborate on the problem you're trying to address? If would be ideal if you could share code for us to reproduce it internally, and verify that all platforms work as intended.

Reply:
This is only applied to the 'mac' or OS X platform. The other platforms use different methods. The issue is that it is not at all working under OS X using clion/cmake. At all. All images in all sample-sessions are unable to load.

Reply:
I see, thanks. I can reproduce the problem in one of the samples.

I suspect your cmake changes make sense, but that the path handling changes could break our internal builds. We'll have to take a closer look, but will get back to you soon.

For the time being, you can use ImguiSession which doesn't depend on any assets.